packages/banner/src/Banner.story.tsx

```diff
@@ -12,7 +12,7 @@ export default {
   title: 'Components/Banner',
   component: Banner,
   parameters: {
-    default: 'Basic',
+    default: 'WithLink',
```

Comment:
This story has 2 links in it 
<img width="229" alt="Screenshot 2023-04-24 at 1 14 05 PM" src="https://user-images.githubusercontent.com/2414030/234068933-c48535cc-1f73-486f-987d-0793225e636e.png">